Makefile

```diff
@@ -111,14 +126,23 @@ fetch-cni-bins: dist/flannel dist/loopback dist/host-local dist/portmap
 
 # For now we've got a checked in version of the plugin.  Eventually
 # this should pull from upstream releases.
+ifeq ($(ARCH), amd64) 
 dist/portmap:
 	mkdir -p dist
 	$(CURL) -L https://github.com/projectcalico/cni-plugin/releases/download/v1.9.0/portmap -o $@
 	chmod +x dist/portmap
 
 dist/flannel dist/loopback dist/host-local:
 	mkdir -p dist
-	$(CURL) -L --retry 5 https://github.com/containernetworking/cni/releases/download/$(CNI_VERSION)/cni-amd64-$(CNI_VERSION).tgz | tar -xz -C dist ./flannel ./loopback ./host-local
+	$(CURL) -L --retry 5 https://github.com/containernetworking/cni/releases/download/$(CNI_VERSION)/cni-$(ARCH)-$(CNI_VERSION).tgz | tar -xz -C dist ./flannel ./loopback ./host-local
+endif
+
+ifeq ($(ARCH), arm64) 
```

Comment:
@caseydavenport Thanks a lot for your review.
Yes, I also think if amd64 and arm64 can use a single version of cni-plugins, which would simplify the work. Actually I have written a version of Makefile change with the same CNI_VERSION=v0.6.0-rc1 and it works, but for the concern of keeping x86 as the original as possible, I separate the processings for x86 and arm64. 
If you think a single version is better and given a certain version, I would merge the separate processings of the 2 archs.

Btw, I would also update my arm64 enabling work on other Calico components in the near future.  
Hope you can give me more help.